modules/social_features/social_user_export/src/Plugin/UserExportPluginBase.php

```diff
@@ -215,25 +214,21 @@ public function profileGetTaxonomyFieldValue($field_name, ProfileInterface $user
       return '';
     }
 
-    $concatenated_value = '';
-    try {
-      $values = $user_profile->get($field_name)->getValue();
-      if (is_array($values) && !empty($values)) {
-        $taxonomy_terms = [];
-        foreach ($values as $x => $value) {
-          if (isset($value['target_id'])) {
-            /** @var \Drupal\taxonomy\Entity\Term $term */
-            $taxonomy_term = Term::load($value['target_id']);
-            $taxonomy_terms[] = $taxonomy_term->getName();
-          }
-        }
-        $concatenated_value = implode(', ', $taxonomy_terms);
-      }
+    /** @var \Drupal\Core\Field\EntityReferenceFieldItemListInterface $field */
+    $field = $user_profile->get($field_name);
+
+    if ($field->isEmpty()) {
```

Comment:
This isn't needed as `referencedEntities` already does this check for us and returns an empty array when the field is empty.